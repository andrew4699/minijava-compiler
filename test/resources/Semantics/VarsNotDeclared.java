class MyClass {
  public static void main(String[] args) {
    System.out.println(new YourClass().yourFunc());
  }
}

class YourClass {
  int a;
  int b;

  public int yourFunc() {
    a = 100;
    b = 12;
    c = 20;
    return new YourClass().yourFunc();
  }
}