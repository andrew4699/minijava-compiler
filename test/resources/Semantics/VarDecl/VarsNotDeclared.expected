[VarDeclVisitor - 14]: c undeclared at YourClass.yourFunc
[VarDeclVisitor - 14]: c undeclared at YourClass.yourFunc
